libs/common/src/sql-db-connection/migrations/1724342914301-AddFeeRecordReconciliationDataColumns.ts

```diff
@@ -0,0 +1,25 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddFeeRecordReconciliationDataColumns1724342914301 implements MigrationInterface {
+  name = 'AddFeeRecordReconciliationDataColumns1724342914301';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            ALTER TABLE "FeeRecord"
+            ADD "reconciledByUserId" nvarchar(255)
+        `);
+    await queryRunner.query(`
+            ALTER TABLE "FeeRecord"
+            ADD "dateReconciled" datetime2
```

Comment:
Do we need a `DEFAULT NULL` here? I am not familiar with this data structure or migration, sorry if this is irrelevant.

Reply:
SQL automatically defaults nullable columns to NULL